R/plotting.R

```diff
@@ -128,9 +128,12 @@ BarplotChainRegion <- function(
 
   plot.data[[data.column]] <- factor(plot.data$family, levels = families)
 
-  if (is.null(cols)) {
-    cols <- GetCategoricalColorPalette(plot.data$group)
+  if (!is.null(cols)) {
+      if (cols == "Colorblind") {
+          cols <- c("#FDE725","#423C81")
+      }
```

Comment:
Could this case be captured in `GetCategoricalColorPalette()`? Its easier if everything related to colors/palettes is in 1 general spot instead of having some weird edge cases handled in other places of the package